Exec/RegTests/HIT/tests/hit-1/hit-1.i

```diff
@@ -57,7 +57,7 @@ amr.plot_vars  =  density Temp
 amr.derive_plot_vars = x_velocity y_velocity z_velocity magvel magvort pressure
 
 # PROBLEM PARAMETERS
-prob.iname = "hit-1.ic"
+prob.iname = "@CMAKE_SOURCE_DIR@/Exec/RegTests/HIT/hit_ic_4_32.dat"
```

Comment:
I get that this makes is to we don't have to copy files. But it also means that the only way to run the case is through `ctest` right? Bringing this up because sometimes I run these cases outside of `ctest`...

Reply:
You have to do `prob.iname = "hit_ic_4_32.dat"` on the command line, which is annoying, but the 1MB dodecane PMF file made me realize we shouldn't keep copies around. I tried symlinks but they require the developer to list the files for each test that need to be symlinked. I guess the best option might be to put the test input files back into the top level executable directory, but that poses some other issues. We should probably talk about the best way to do this next week.

Reply:
yeah sounds like there's no perfect solution. let's talk